parachain/runtime/interlay/src/constants.rs

```diff
@@ -0,0 +1,41 @@
+//! A set of constant values used in the kintsugi runtime.
+
+/// Money matters.
+pub mod currency {
+    pub use primitives::{Balance, CurrencyId, CurrencyId::Token, DOT, IBTC, INTR};
+
+    pub const NATIVE_CURRENCY_ID: CurrencyId = Token(INTR);
+    pub const PARENT_CURRENCY_ID: CurrencyId = Token(DOT);
+    pub const WRAPPED_CURRENCY_ID: CurrencyId = Token(IBTC);
+
+    // https://github.com/paritytech/polkadot/blob/c4ee9d463adccfa3bf436433e3e26d0de5a4abbc/runtime/polkadot/src/constants.rs#L18
+    pub const UNITS: Balance = NATIVE_CURRENCY_ID.one();
+    pub const DOLLARS: Balance = UNITS; // 10_000_000_000
+    pub const CENTS: Balance = DOLLARS / 100; // 100_000_000
+    pub const MILLICENTS: Balance = CENTS / 1_000; // 100_000
+
+    pub const fn deposit(items: u32, bytes: u32) -> Balance {
+        items as Balance * 20 * DOLLARS + (bytes as Balance) * 100 * MILLICENTS
+    }
+}
+
+/// Time and blocks.
+pub mod time {
```

Comment:
Would it make sense to put this module in a separate crate that is used by all parachain runtimes? I would like to investigate ways in the future to share more of the configs among the different chains, this could be a first step in that direction

Reply:
For now I just want to split from the `runtime/lib.rs` to be more idiomatic with other project structures. Eventually we'll adopt a `runtime-common` crate were shared constants can live but some of these (particularly the currencies) are runtime dependent.

Reply:
Yea that's why I commented on the `time` module rather than the currency module